considerations.mkd

```diff
@@ -58,3 +58,38 @@ precision than necessary.
    represents edges as great circles will have different interpretations
    of edges, which vary more the longer the edges are. Try to
    avoid edges of more than 180 degrees in length as far as possible.
+
+
+# Extensibility
+
+In GeoJSON, an "extension" is any property not defined in section 2.
+Consuming implementations that encounter unfamiliar extensions MUST NOT
+stop processing or signal an error and MUST continue processing the
+items as if those properties were not present. Note that support for
+extensions can vary across implementations and no normative processing
+model for extensions is defined. Accordingly, implementations that rely
+too heavily on the use of extensions may experience reduced
+interoperability with other implementations.
+
+## GeoJSON types are fixed
+
+Implementations MUST NOT extend the fixed set of GeoJSON types:
+FeatureCollection, Feature, Point, LineString, MultiPoint, Polygon,
+MultiLineString, MultiPolygon, and GeometryCollection.
+
+## Semantics of GeoJSON properties are fixed
+
+Implementations MUST NOT change the the semantics of GeoJSON properties.
+
```

Comment:
i know that we discussed this on the geojson list a while back (both in the context of "time" in may 2013, and in the context of GPX).

in the case of `Point M` simply defines that the third coordinate is "moment" - i don't see that as changing the semantics of `Point`, as a point is noted as 2 (or more) entries, leaving additional elements beyond longitude and latitude as intentionally undefined.

the *point* (pun intended) of my comment is to note that sometimes extending the value of a GeoJSON object is sometimes optimal, and am looking for a clean way of doing it, rather than being explicitly disallowed.

i think i already saw a couple of notes on the draft mailing list, but if you feel i should propose it, i am happy to.